lib/docs/src/switcher.stories.tsx

```diff
@@ -53,7 +55,46 @@ export const Primary = (props: Props): JSX.Element => {
 	);
 };
 
-Utils.fixPrimary(Primary);
+export const Basic = (): JSX.Element => {
+	const [value, setValue] = useState<number>(0);
+	const options: SwitcherOption<number>[] = [
+		{ value: 0, label: "Left" },
+		{ value: 1, label: "Center" },
+		{ value: 2, label: "Right" },
+	];
+	return <Switcher value={value} setValue={setValue} options={options} />;
+};
+
+Utils.desc(Basic)(`
+Switchers are [controlled][1] components. You should maintain a [state][2] for
+the current value, and pass the control to a Switcher. The type of your state
+can be anything since Switcher is a [generic][3] component.
+
+The options for a Switcher are defined via its \`options\` prop. They require
+the \`value\` and \`label\` attributes. See the "SwitcherOption" table below
+for the complete interface of an option.
+
+[1]: https://reactjs.org/docs/forms.html#controlled-components
+[2]: https://reactjs.org/docs/hooks-state.html
+[3]: https://www.typescriptlang.org/docs/handbook/2/generics.html
+`);
+
+export const Toggle = (): JSX.Element => {
+	const [on, setOn] = useState<boolean>(true);
+	const options: SwitcherOption<boolean>[] = [
+		{ value: true, label: "On" },
+		{ value: false, label: "Off" },
+	];
+	return <Switcher highlight value={on} setValue={setOn} options={options} />;
+};
+
+Utils.desc(Toggle)(`
+Switchers can be used for \`boolean\` values. In these cases, they work like
+[Toggles][1], letting users flip between on-off states. The \`highlight\` prop is often
+used in such cases to make the selected option more prominent.
+
+[1]: https://www.nngroup.com/articles/toggle-switch-guidelines/
+`);
```

Comment:
@dangbt this part is for you :)) the doc for https://github.com/moaijs/moai/pull/145